crates/astria-composer/src/composer.rs

```diff
@@ -22,12 +51,22 @@ use crate::{
 /// Composer is a service responsible for submitting transactions to the Astria
 /// Shared Sequencer.
 pub struct Composer {
-    /// ApiServer is used for monitoring status of the Composer service.
+    /// `ApiServer` is used for monitoring status of the Composer service.
     api_server: ApiServer,
-    /// Searcher establishes connections to individual rollup nodes, receiving
+    /// `Searcher` establishes connections to individual rollup nodes, receiving
     /// pending transactions from them and wraps them as sequencer transactions
     /// for submission.
     searcher: Searcher,
+    /// `ExecutorHandle` to communicate SequenceActions to the Executor
+    /// This is at the Composer level to allow its sharing to various different collectors.
+    executor_handle: ExecutorHandle,
+    /// `GrpcCollectorListener` is the tcp connection on which the gRPC collector is running
+    grpc_collector_listener: TcpListener,
+}
+
+#[derive(Clone)]
+struct ExecutorHandle {
+    send_bundles: Sender<SequenceAction>,
```

Comment:
```suggestion
    sequence_action_tx: Sender<SequenceAction>,
```

since this sends individual actions not bundles

Reply:
this is done